File: D11_APP/function/bluetooth/src/main/java/com/luxshare/bluetooth/listener/IReadListener.java
package com.luxshare.bluetooth.listener;

/**
 * The bluetooth read listener.
 *
 * @author ChenCe
 * @version version
 */
public interface IReadListener {

    void onReadSuccess(byte[] readBuffer); // 读成功

    void onReadFailure(); // 读失败
}

File: D11_APP/function/ble/src/main/java/com/luxshare/ble/interfaces/IReadListener.kt
package com.luxshare.ble.interfaces

import com.luxshare.ble.eventbus.EventBusMessage

/**
 * The bluetooth read listener.
 *
 * @author ChenCe
 * @version version
 */
interface IReadListener {
    fun onReadSuccess(readBuffer: ByteArray) // 读成功
    fun onReadFailure() // 读失败
    fun onReadEventMessage(eventBusMessage: EventBusMessage) {
    }
}
